test/rules/arrow-parens/test.ts.lint

```diff
@@ -19,9 +19,14 @@ var barbarbar = <T>(method: (a) => T) => {
 var piyo = <T, U>(method: () => T) => {
     method();
 };
+const validAsync = async (param: any) => {};
+const validAsync = async (param) => {};
 
 // invalid case
 var e = (a => {})(1);
          ~            [Parentheses are required around the parameters of an arrow function definition]
 var f = a => {};
         ~             [Parentheses are required around the parameters of an arrow function definition]
+
+// I think should warn following case, but I have no idea to validate this. 
+const invalidAsync = async param => {};
```

Comment:
Actually, I don't think `async param => {}` is even valid TypeScript. Can you just delete this test?